app/lib/spectre_form_builder.rb

```diff
@@ -137,6 +138,24 @@ def date_field(method, options = {})
     end
   end
 
+  # Render a set of fields for every object in an association, along with a
+  # button to create new instances of the join model.
+  #
+  # Based on the "Stimulus" example from this absolute legend:
+  # https://stackoverflow.com/a/71715794
```

Comment:
Wow that's one of the most thorough SO posts I've seen in a while. 🤩

Reply:
It must have taken soooo long to write up!